sky/skylet/providers/lambda_labs/lambda_utils.py

```diff
@@ -0,0 +1,161 @@
+"""Lambda Labs helper functions."""
```

Comment:
nit: It would be nice to have type annotations for the functions

Reply:
Done.